lambda_functions/notification_service/resources/mail_templates/pubsub/de/level_up/text.txt

```diff
@@ -0,0 +1 @@
+Herzlichen Glückwunsch {name}! Du bist aufgestiegen und hast nun Level {level} erreicht. Weiter so! 
```

Comment:
Macht es vielleicht Sinn hier einen extra Ordner zu erstellen (z.B. `pubsub_templates`) anstatt in `mail_templates` zu gehen?